### ISSUE TITLE
Allow changing verbiage of claim success on CoC acceptance

### ISSUE THREAD

**Issue by itsthejoker** [GrafeasGroup/tor#102]
Currently when a new user comes on board, they go through the code of conduct and that simply calls the rest of the claim code. The current message reads,

```
The post is yours! Best of luck and thanks for helping!

Please respond with "done" when complete so we can check this one off the list!
```

Let's add a flag to that so we know if we're coming from the code of conduct so we can tweak that message to make it more friendly and tell people to talk to the mods if they get stuck. Maybe something like,

```
The post is yours! Best of luck and thanks for helping!

Since this is your first one, please make sure to message the mods if you have any questions about the process (and / or join our [Discord](link) to talk to other volunteers). We're here to help!

Please respond with "done" when complete so we can check this one off the list!
```

**Comment by itsthejoker** [comment]
And maybe mention the formats again? From @pejmanpoh

**Comment by PejmanPoh** [comment]
I might take a crack at this to get acquainted with the code base. Seems like tor_app is a long way from seeing any code. Also a good chance to start using python anyway.

**Comment by DavidArchibald** [comment]
As a (very) new member of /r/ToR, this idea seems like a good idea to help new transcribers along. The mod support offered should help, plus the discord link is even better as I missed it in my reading of the sidebar, likely because I skimmed over some of the first parts though... it seemed pretty standard. (I think other subreddits I've seen have have put their discord link lower on the sidebar and more bold, but it's probably my fault.)

When I came into the subreddit, I was a bit unsure what to do, even with the template and wiki pages. Ex. Remove the square brackets or not(they have meant insertion in almost every other non coding thing I've done).

My point in sharing that is that I didn't think I could do anything, but hope that I wasn't about to be temp banned for 7 days for messing up the formatting(which I genuinely thought could happen, but was soon dispelled). 

So while in my case a nice mod came and verified I'd done it okay, telling the new redditors they can contact mods could help.
